package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "microstates",
-  "version": "0.7.3",
+  "version": "0.8.0",
   "description": "Graceful state transitions for data of any shape.",
   "keywords": [
-    "blood",
+    "statemachines",
```

Comment:
let's add `lens` as a keyword while we're at it. And should it be `state-machine`?

Reply:
Sorry, I missed this. I'll add it in the README branch.